ffxivbot/views/tata.py

```diff
@@ -149,8 +149,8 @@ def tata(req):
             coolq_edition = coolq_edition[0].upper() + coolq_edition[1:]
         friend_list = json.loads(bot.friend_list)
         friend_num = (
-            len(friend_list["friends"])
-            if friend_list and "friends" in friend_list.keys()
+            len(friend_list)
+            if friend_list
```

Comment:
Can you make the code compatible with the current database schema?

Reply:
The friend_list is array now because new api response is a JSON array.